ceph/ceph_admin/apply.py

```diff
@@ -30,3 +41,51 @@ def apply(self, role, command, placements):
                 daemon=role,
                 ids=placements,
             )
+
+    def orch_remove(self, **config):
+        """
+        Remove service using "ceph orch rm <svc-id>" command
+        Args:
+            config: service remove arguments
+
+        config:
+            service_name: service name
+            check_status: check command status
+        """
+        service_name = config.get("service_name")
+        check_status = config.get("check_status", True)
+
+        # check service existence
+        if not (service_name or self.get_role_service(service_name=service_name)):
```

Comment:
IMO, we can avoid this statement. Without this check, the upper layers can pass negative results to validate its behavior.

Reply:
service_name validation is taken care above, 
``` 
def orch_remove(svc_name, check_status=True):
```